.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -327,8 +546,57 @@ dynamodb_DeleteItem:
             - description:
               snippet_tags:
                 - dynamodb.rust.delete-item
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-table-read-write.html#dynamodb-example-table-read-write-deleting-an-item
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createclientv3
+            - description: Delete the item.
+              snippet_tags:
+                - dynamodb.JavaScript.item.deleteItemV3
+        - sdk_version: 2
+          github: javascript/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v2/developer-guide/dynamodb-example-table-read-write.html#dynamodb-example-table-read-write-deleting-an-item
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.JavaScript.item.deleteItem
   services:
     dynamodb: {DeleteItem}
+dynamodb_Delete:
+  title: Delete an item from a &DDB; table using the &DDB; document client using an &AWS; SDK
+  title_abbrev: Delete an item from a table using the document client
+  synopsis: delete an item from a &DDB; table using the &DDB; document client.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-dynamodb-utilities.html#dynamodb-example-document-client-delete
+          excerpts:
+            - description: Create the &DDB; document client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createdocclientv3
+            - description: Create the &DDB; client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createclientv3

```

Comment:
Should this go first? The code reads better that way for me, but it's up to you.

Reply:
yes, fixed